src/standalone/Soil/energy_hydrology.jl

```diff
@@ -970,9 +962,22 @@ function soil_turbulent_fluxes_at_a_point(
     else
         Ẽ *= 0 # condensation, set to zero
     end
-    H_liq::FT = -ρ_air * cp_m * ΔT / r_ae_liq
 
-    # Sublimation:
+    # sensible heat flux
+    SH_liq = SurfaceFluxes.sensible_heat_flux(
```

Comment:
For the soil model, we call SF twice - once for ice, and once for water. We then combine the fluxes from them to get the total SH, LH. the vapor fluxes are applied to ice and water differently, so they are kept separate.

For ice, we supply a beta factor and do not need an additional surface resistance, and so we use the output of `surface_conditions` directly. For water, we do, and it's a little complicated how we handle it. But once we correct E, we compute SH and LH the same as SF.jl does.

Reply:
Note that for both we do not allow condensation/frost. We can/should revisit this.

Reply:
it would be nice to add this (or part of it) as comments in the code

Reply:
Ill addit!